generic/tests/vlan.py

```diff
@@ -186,6 +186,8 @@ def get_netkvmco_path(session):
     login_timeout = int(params.get("login_timeout", 360))
     prepare_netkvmco_cmd = params.get("prepare_netkvmco_cmd")
     set_vlan_cmd = params.get("set_vlan_cmd")
+    driver_name = params["driver_name"]
```

Comment:
`driver_name` is not used, so only `driver_verifier` is needed in cfg and py.
Some other changes have the same problem.

Reply:
Okay,will update the code.

Reply:
Hi @PaulYuuu , `driver_name` is called by `driver_verifier = params.get("driver_verifier", driver_name)`. When the Windows version does not need to add "ndis" to `driver_verifier`, `driver_verifier` will use `driver_name` as the default value.

Reply:
But you can handle all windows guests with a single parameter. For your change in vlan.cfg:
```
        driver_name = netkvm
        driver_verifier = ${driver_name}
        Win2016, Win2019, Win8..1, Win2012..r2:
                driver_verifier += " ndis"
```
You just need to set `driver_verifier = netkvm`, so the default value is `netkvm`, the driver_name is not necessary for driver verification.

Reply:
But you can handle all windows guests with a single parameter. For your change in vlan.cfg:
```
        driver_name = netkvm
        driver_verifier = ${driver_name}
        Win2016, Win2019, Win8..1, Win2012..r2:
                driver_verifier += " ndis"
```
You just need to set `driver_verifier = netkvm`, so the default value is `netkvm`, the driver_name is not necessary for driver verification.

Reply:
Okay, you are right, thanks for your patience.